migrations/versions/20221111_06f24cc01c09_add_acoustic_field.py

```diff
@@ -0,0 +1,84 @@
+"""Add Acoustic field
+
+Revision ID: 06f24cc01c09
+Revises: 406251207612
+Create Date: 2022-11-11 17:32:04.411812
+
+"""
+# pylint: disable=no-member invalid-name
+# no-member is triggered by alembic.op, which has dynamically added functions
+# invalid-name is triggered by migration file names with a date prefix
+# invalid-name is triggered by top-level alembic constants like revision instead of REVISION
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '06f24cc01c09'  # pragma: allowlist secret
+down_revision = '406251207612'  # pragma: allowlist secret
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    acoustic_field_table = op.create_table('acoustic_field',
```

Comment:
Since this has moved from being in memory to a database call and we seem to be querying it frequently (I think correct me here if I am wrong). Should we be concerned with database performance or adding any indexes here?

Reply:
Thanks for the reminder.
We fetch the whole content of the table once when the `ctms/bin/acoustic_sync.py` script is executed. 
Hence we're good here

Reply:
Great thank you!